src/ts/components/seekbar.ts

```diff
@@ -138,18 +166,39 @@ export class SeekBar extends Component<SeekBarConfig> {
     // (the call must be up here to be executed for the volume slider as well)
     this.setPosition(this.seekBarBackdrop, 100);
 
+    // Add seekbar controls to the seekbar
+    const seekBarController = new SeekBarController(this.config.keyStepIncrements, player, uimanager.getConfig().volumeController);
+
+    seekBarController.setSeekBarControls(this.getDomElement(), this.getSeekBarType);
+
     if (!configureSeek) {
+      this.seekBarType = SeekBarType.Volume;
```

Comment:
Wouldn't that fit better in the subclass (Volume Slider)?

Reply:
Yes but I wanted to keep everything in one place regarding Web Accessibility since the `aria` attributes for the volume are defined here. In the `Volumeslider` it is not that easy to add those labels

Reply:
👍